avaframe/com1DFA/com1DFA.py

```diff
@@ -1399,6 +1297,26 @@ def computeEulerTimeStep(cfg, particles, fields, dt, dem, Tcpu, frictType):
     tcpuPos = time.time() - startTime
     Tcpu['Pos'] = Tcpu['Pos'] + tcpuPos
 
+    # Split particles
+    if cfg.getint('splitOption') == 0:
+        # split particles with too much mass
+        # this only splits particles that grew because of entrainment
+        particles = particleTools.splitPartMass(particles, cfg)
+    elif cfg.getint('splitOption') == 1:
+        # split merge operation
+        # first update fields (compute grid values) because we need the h of the particles to get the aPart
+        # ToDo: we could skip the update field and directly do the split merge. This means we would use the old
+        # h. No big deal I think
```

Comment:
?? needed?

Reply:
This is an open question, I do not have the answer

Reply:
ah, then at least remove the 'no big deal...' part